edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/IRegistryOnBehalfOfApiClient.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Http
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    public interface IRegistryOnBehalfOfApiClient
+    {
+        Task<RegistryApiHttpResult> PutModuleAsync(string actorEdgeDeviceId, CreateOrUpdateModuleOnBehalfOfData requestData, string ifMatchHeader);
```

Comment:
Nit: Does this have to be an Edge Device?

Reply:
RegistryOnBehalfOfApiClient.cs